index.html

```diff
@@ -4,131 +4,162 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js" defer></script>
+    <title>SmartBuild Main Conference</title>
+    <link rel="stylesheet" href="css/style.css">
+    <script src="javascript/script.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap" rel="stylesheet"> 
 </head>
 <body>
 
+    <div class="social-icons-section">
```

Comment:
- [ ] As the previous reviewer suggested kindly follow the design provided, e.g: the top bar should be darker

|Your design|Provided design|
|-|-|
|![Screenshot_20221022_154126](https://user-images.githubusercontent.com/25227022/197345319-6ab43228-897e-4836-95ff-d2482191ad13.png)|![](https://user-images.githubusercontent.com/20567503/197232810-e2c05c86-94a1-4593-8359-3274420e51dc.png)|